mne/report/report.py

```diff
@@ -278,6 +279,41 @@ class _ContentElement:
     html: str
 
 
+def _check_tags(tags) -> Tuple[str]:
+    # Must be iterable, but not a string
+    if (isinstance(tags, str) or not isinstance(tags, (Sequence, np.ndarray))):
+        raise TypeError(
+            f'tags must be a collection of str, but got {type(tags)} '
+            f'instead: {tags}'
+        )
+    tags = tuple(tags)
+
+    # Check for invalid dtypes
+    bad_tags = [tag for tag in tags
+                if not isinstance(tag, str)]
+    if bad_tags:
+        raise TypeError(
+            f'tags must be strings, but got the following instead: '
+            f'{", ".join([str(tag) for tag in bad_tags])}'
+        )
```

Comment:
```suggestion
    for ti, tag in tags:
        _check_option(f'tags[{ti}]', tag, str)
```

Reply:
Here again I have the issue that `_check_option()` immediately raises if the condition is not met. But I want to collect all invalid tags and tell the user, so they don't have to start fixing them one after the other after each new run which shows a problem with yet another tag